locale/fi/icons.json

```diff
@@ -1,9 +1,14 @@
 {
   "title": "Ikonit (Icons)",
   "intro": "Tältä sivulla löydät kaikki ikonit jotka ovat käytössä suomi.fi palveluissa. Ikonien käyttöohjeistukset löydät ikonien komponentti sivulta.",
```

Comment:
Not really in the scope of this PR, but might be worth changing nevertheless. "ikonien komponentti sivulta" just hurts my soul a bit. Would "ikonikomponentin sivulta" or something along those lines work better grammatically?

If so, then the `iconPageLink.title` could be similar, as in "Ikonikomponentin käyttöohjeistus".

Reply:
Those suggestions sound better so I changed the localisations.